src/components/PeopleTable/PeopleTable.tsx

```diff
@@ -0,0 +1,85 @@
+import { useParams, useSearchParams } from 'react-router-dom';
+import classNames from 'classnames';
+import { Person, SearchParams } from '../../types';
+import { PeopleTableItem } from '../PeopleTableItem';
+import { TABLE_ATTRIBUTES } from '../../utils/constants';
+import { SearchLink } from '../SearchLink';
+
+interface Props {
+  people: Person[];
+}
+
+export const PeopleTable: React.FC<Props> = ({ people }) => {
+  const { selectedPersonSlug = '' } = useParams();
+  const [searchParams] = useSearchParams();
+  const sort = searchParams.get('sort') || '';
+  const order = searchParams.get('order') || '';
+
+  const handleAddSort = (attribute: string): SearchParams => {
+    switch (sort === attribute) {
+      case true: {
+        if (order) {
+          return { order: null, sort: null };
+        }
+
+        return { order: 'desc' };
+      }
+
+      case false:
+      default: {
+        return { order: null, sort: attribute };
+      }
+    }
```

Comment:
Maybe better to use just `if` in this case?
We have only 2 options `true` and `false`, it looks somehow strange